http-server/server.py

```diff
@@ -72,6 +72,20 @@ def event(self, title, text):
         self.stats.flush()
 
 
+class NoopMetrics:
```

Comment:
Is dit bewuste naamgeving of moest dit nog anders?

Reply:
Ik had een implementatie van een "metrics" class nodig die niets deed, dus NoOp. Of dit de beste manier is kan ik je niet vertellen.